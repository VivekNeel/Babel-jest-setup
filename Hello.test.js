import React from "react";
import Link from "./Hello";
import renderer from "react-test-renderer";

jest.mock('react-native-gesture-handler', () => {});


it("renders correctly", () => {
      const tree = renderer
        .create( < Link page = "http://www.facebook.com" > Facebook < /Link>)
          .toJSON(); expect(tree).toMatchSnapshot();
        });