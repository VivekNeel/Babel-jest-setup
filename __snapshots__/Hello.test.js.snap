// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  pointerEvents="box-none"
  style={
    Object {
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 20,
    }
  }
>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</View>
`;
